doc/Doxyfile.in

```diff
@@ -134,7 +134,7 @@ HTML_FILE_EXTENSION    = .html
 HTML_HEADER            = @CMAKE_CURRENT_SOURCE_DIR@/header.html
 HTML_FOOTER            = @CMAKE_CURRENT_SOURCE_DIR@/footer.html
 HTML_STYLESHEET        =
-HTML_EXTRA_STYLESHEET  = @CMAKE_CURRENT_SOURCE_DIR@/stylesheet.css
+HTML_EXTRA_STYLESHEET  = @CMAKE_CURRENT_SOURCE_DIR@/doxygen-awesome.css
```

Comment:
BTW, no miracle happened, [this page](http://pullrequest.opencv.org/buildbot/export/pr/21065/docs/d4/d1c/namespacecv_1_1gapi.html) is still look broken on FHD resolution (1920x1080).

Reply:
Perhaps complex types like `std::enable_if< cv::detail::accepted_infer_types< Input >::value, cv::GInferListOutputs >::type`  should be replaced with shorter aliases using cosmetic typedefs.